src/components/Rebuild/Rebuild.scss

```diff
@@ -0,0 +1,11 @@
+.rebuild-btn {
+  position: absolute !important;
```

Comment:
avoid `!important`, instead prefer more specific selectors

`.bx--btn.rebuild-btn` or similar


also avoid `position: absolute` if possible as it can cause problems with responsive layouts, leading to overlapping content and other issues